pkg/route/ingress/ingress.go

```diff
@@ -38,6 +38,8 @@ import (
 	routeclient "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
 	routeinformers "github.com/openshift/client-go/route/informers/externalversions/route/v1"
 	routelisters "github.com/openshift/client-go/route/listers/route/v1"
+
+	s "strings"
```

Comment:
Typically we import standard go packages at the top of the import block using their default name (and it doesn't look like there's any name collisions with a different strings package here).

Reply:
done